Automattic/wp-calypso
# Show Subscription Type column in Subscribers Page

Closes https://github.com/Automattic/wp-calypso/issues/78182

## Proposed Changes

The changes in this PR show the Subscription Type column on the Subscribers page. Currently, it shows all the paid subscriptions of the user, or "Free tier" if there is no paid subscription.

<img width="1135" alt="Screenshot 2023-06-14 at 19 17 42" src="https://github.com/Automattic/wp-calypso/assets/3832570/f2fe6ae1-ab32-4b11-b785-c80b8d824f0d">


## Testing Instructions

You will need to perform this test with a website that has paid users.

1. Apply this PR and start the application.
2. Go to this URL: http://calypso.localhost:3000/subscribers/[the-domain-of-your-site]. You should see the subscription plans of your subscribers listed in the Subscription Type column.
